frontend/app/lib/components/DemoLevel.jsx

```diff
@@ -0,0 +1,553 @@
+/*
+Auto-generated by: https://github.com/pmndrs/gltfjsx
+Command: npx gltfjsx@6.2.3 frontend/app/public/demolevel.glb
```

Comment:
Bin nicht so ein freund von Webung im code :D Dann lieber in die readme packen oder so